Phonebook.Frontend/src/app/shared/dialogs/user-information/user-information.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit, Inject } from '@angular/core';
+import { CurrentUserService } from 'src/app/services/api/current-user.service';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material';
+import { DialogData } from 'src/app/shared/components/user/user-detail/user-detail.component';
+
+@Component({
+  selector: 'app-user-information',
+  templateUrl: './user-information.component.html',
+  styleUrls: ['./user-information.component.scss']
+})
+export class UserInformationComponent implements OnInit {
+
+  public currentUserId: string = '';
+  constructor(
+    public currentUserService: CurrentUserService,
+    public dialogRef: MatDialogRef<UserInformationComponent>,
+    @Inject(MAT_DIALOG_DATA) public data: DialogData | any
+  ) {}
+
+  public ngOnInit() {
+    this.currentUserService.getCurrentUserId().subscribe(
+      id => {
+        this.currentUserId = id;
+      },
+      error => {
+        // do nothing, as the id will never be ''
+      }
+    );
+  }
+  public get onMyProfile(): boolean {
```

Comment:
I think we have this function somewhere else aswell. Would be good to refactor into one.

Reply:
also didn't found it